tools/lsp/completion.rs

```diff
@@ -301,14 +300,17 @@ pub(crate) fn completion_at(
     } else if node.kind() == SyntaxKind::Document {
         let r: Vec<_> = [
             // the $1 is first in the quote so the filename can be completed before the import names
-            ("import", "import { $2 } from \"$1\";"),
-            ("component", "component $1 {}"),
-            ("struct", "struct $1 {}"),
-            ("global", "global $1 {}"),
-            ("export", "export { $1 }"),
-            ("export component", "export component $1 { }"),
-            ("export struct", "export struct $1 {}"),
-            ("export global", "export global $1 {}"),
+            ("import", "import { $1 } from \"${2:std-widgets.slint}\";"),
+            ("component", "component ${1:Component} ${2:inherits ${3:Rectangle} }{\n    $0\n}"),
```

Comment:
I don't think se should have inherits by default.  We should encourage not to.   And especially not of Rectangle.

Reply:
I still feel like we shouldn't suggest it

Reply:
I agree with Olivier. This is like rust analyzer offering snippets that clippy would cry about. Inherits should be a very conscious choice.